sr_robot_launch/scripts/sr_ur_arm_unlock

```diff
@@ -0,0 +1,166 @@
+                fault = True
+                rospy.loginfo("Fault detected on: %s", arm)
+                rospy.loginfo("This can be caused by using the external e-stop. Have you released it?")
+                self.call_arm_service(arm, "restart_safety", Trigger)
+        return fault
+
+    def check_arms_robot_mode(self, arms):
+        arm_needs_starting = False
+        for arm in arms:
+            robot_mode = self.call_arm_service(arm, "get_robot_mode", GetRobotMode)
+            if robot_mode.robot_mode.mode == RobotMode.IDLE or robot_mode.robot_mode.mode == RobotMode.POWER_OFF:
+                rospy.loginfo("Starting arm: %s", arm)
+                arm_needs_starting = True
+        return arm_needs_starting
+
+    def clear_arms_popups(self, arms):
+        for arm in arms:
+            self.call_arm_service(arm, "close_safety_popup", Trigger)
+        rospy.sleep(2)
+        for arm in arms:
+            self.call_arm_service(arm, "close_popup", Trigger)
+        rospy.sleep(2)
+
+    def check_fix_program_loaded_arms(self, arms):
+        sleep_time = False
+        for arm in arms:
+            if not rospy.get_param("/" + arm + "_sr_ur_robot_hw/headless_mode"):
+                play_msg = self.call_arm_service(arm, "program_state", GetProgramState)
+                if play_msg.program_name == "null":
+                    rospy.loginfo("Not in headless mode. Loading program: %s for arm: %s", self.external_control_program_name, arm)
+                    resp = self.call_arm_service(arm, "load_program", Load)
+                    rospy.loginfo("%s", resp)
+                    sleep_time = True
+        return sleep_time
+
+    def check_fix_program_playing_arms(self, arms):
+        sleep_time = False
+        for arm in arms:
+            if not rospy.get_param("/" + arm + "_sr_ur_robot_hw/headless_mode") :
+                play_msg = self.call_arm_service(arm, "program_state", GetProgramState)
+                if play_msg.state.state == ProgramState.STOPPED or play_msg.state.state == ProgramState.PAUSED:
+                    rospy.loginfo("Not in headless mode. Starting program: %s for arm: %s", play_msg.program_name, arm)
+                    self.call_arm_service(arm, "play", Trigger)
+                    sleep_time = True
+            else:
+                rospy.loginfo("Headless mode detected, resending robot program to arm: %s", arm)
+                self.call_arm_service(arm, "resend_robot_program", Trigger, dashboard=False)
+                rospy.sleep(2)
+                self.call_arm_service(arm, "resend_robot_program", Trigger, dashboard=False)
+                sleep_time = True
+        return sleep_time
+
+    def release_arm(self):
+        try:
+            rospy.loginfo("Unlock arm(s) signal received. Checking arms.")
+            rospy.loginfo("Checking e-stops ...")
+            self.check_arms_e_stops(self.arms)
+            rospy.loginfo("Checking protective stops ...")
```

Comment:
checking and releasing

Reply:
Checking is echo'd here, releasing is echoed in the function if it needs releasing